great_expectations/experimental/datasources/dynamic_pandas.py

```diff
@@ -273,10 +273,20 @@ def _create_pandas_asset_model(
     model = pydantic.create_model(model_name, __base__=model_base, type=type_field, **fields_dict)  # type: ignore[call-overload] # FieldSpec is a tuple
     # can't set both __base__ & __config__ when dynamically creating model
     model.__config__.extra = extra
+
+    def _get_reader_method(self) -> str:
+        return f"read_{self.type}"
+
+    def _get_reader_options_include(self) -> set[str] | None:
+        return None
+
+    setattr(model, "_get_reader_method", _get_reader_method)
+    setattr(model, "_get_reader_options_include", _get_reader_options_include)
```

Comment:
Do we even need this bit of metaprogramming?
Why not just define these methods on the `_FilesystemDataAsset`?

Reply:
@Kilo59 I will try.  One second please.

Reply:
@Kilo59 Upon a closer look, I would prefer an approach where in the `_FilesystemDataAsset`, `Pandas` is not a preferred implementation, but equal to others that utilize filesystems.  Happy to discuss.  Thank you.